### ISSUE TITLE
There is a hard-coded temp record in the database and it does not have a `onsys_fl` designation

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#15220]
> What application are you using?

Vision Zero Editor

> Describe the problem.

There is a hard-coded temp record in the database and it does not have a `onsys_fl` designation. Please update the null value to 'N'.

> Is there anything else we should know?

N/A

> Website Address

https://visionzero.austin.gov/editor/#/crashes/180290542

Internet Browser: Edge

> Requested By
Xavier A.

Request ID: DTS24-111940

**Comment by mddilley** [comment]
@xavierapostol @patrickm02L This is done. I did a test run with a local copy of the production data and app, and then made the updates in the production DB. I checked the crash in VZE and the DB, and I'm seeing the updated values.

Change log entry:
<img width="487" alt="Screenshot 2024-01-26 at 5 11 38 PM" src="https://github.com/cityofaustin/atd-data-tech/assets/37249039/c616cd04-0ba7-4510-9319-353ef1cf7660">
